### PR TITLE
Update image-references with new image names

### PR DESCRIPTION
**Pull request by donpenney**
Signed-off-by: Don Penney <dpenney@redhat.com>

/cc @nishant-parekh

### CODE REVIEW

File: manifests/image-references [hunk 1]
```diff
@@ -2,11 +2,11 @@ kind: ImageStream
 apiVersion: image.openshift.io/v1
 spec:
   tags:
-  - name: cluster-group-upgrades-operator-precache
+  - name: telco-5g-ran4-topology-aware-lifecycle-operator-precache-rhel8
```
Comment (nishant-parekh) [inline manifests/image-references:5]: this should just be the repository name `topology-aware-lifecycle-operator-precache-rhel8`

Comment (nishant-parekh) [inline manifests/image-references]: this should just be the repository name `topology-aware-lifecycle-operator-precache-rhel8`

File: manifests/image-references [hunk 2]
```diff
@@ -2,11 +2,11 @@ kind: ImageStream
...
     from:
       kind: DockerImage
       name: quay.io/openshift-kni/cluster-group-upgrades-operator-precache:4.10.0
-  - name: manager
+  - name: openshift4-topology-aware-lifecycle-manager-rhel8-operator
```
Comment (nishant-parekh) [inline manifests/image-references:9]: this should also just be repository name: `topology-aware-lifecycle-manager-rhel8-operator`

Comment (nishant-parekh) [inline manifests/image-references:9]: this should also just be repository name: `topology-aware-lifecycle-manager-rhel8-operator`

**Review by nishant-parekh — CHANGES_REQUESTED** [review]

File: manifests/image-references (hunk 2 above)
Comment (donpenney) [reply manifests/image-references]: Updated

**Review by nishant-parekh — APPROVED** [review]

File: manifests/image-references (hunk 1 above)
Comment (donpenney) [reply manifests/image-references]: Updated

**Pull request merged**